dist/angular-apimock.js

```diff
@@ -237,7 +263,8 @@ angular.module('apiMock', [])
 
 			// replace apiPath with mockDataPath.
 			var oldPath = req.url;
-			var redirectedPath = req.url.replace(config.apiPath, config.mockDataPath);
+
+			var redirectedPath = req.url.replace(apiPathMatched(req.url, config.apiPath), config.mockDataPath);
```

Comment:
This got me thinking... now all these possible variations of API's will get squashed to one mock data folder. Is that what people want? I haven't had the need for multiple API URL's so I frankly don't know. Would one expect apiPath1 to match mockDataPath1, and apiPath2->mockDataPath2, and so on? Or would they expect apiPath1|apiPath2->mockDataPath?